sw/device/silicon_creator/lib/drivers/retention_sram.h

```diff
@@ -75,7 +87,14 @@ OT_ASSERT_MEMBER_OFFSET(retention_sram_t, owner, 2048);
 OT_ASSERT_SIZE(retention_sram_t, 4096);
 
 enum {
+  /**
+   * ES version.
```

Comment:
Nit: expand the acronym. Is this "extended support"?

Reply:
It actually stands of engineering sample. Sure.